src/app/login/page.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+import React, { useState } from 'react';
+
+const Login: React.FC = () => {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        console.log(email, password);
+    };
+
+    return (
+        <div className="flex items-center justify-center min-h-screen p-4">
+            <form onSubmit={handleSubmit} className="flex flex-col items-center justify-center min-h-screen w-full sm:max-w-md p-8 space-y-6 rounded-lg">
+
+                <div className="w-20 flex flex-col mx-auto justify-center">
+                <img src="/logo.png" alt="Logo" className="mx-auto w-15 h-15 -mt-7" />
+
+                    <h1 className="mx-auto font-bold tracking-wide">OrthoChat</h1>
```

Comment:
Font size for app name needs to be bigger. Check Figma design file, it's `40px`.